Help_Desk_process.md

```diff
@@ -1,44 +1,46 @@
 
-  a. I did the task. (The assignee can then resolve (not close) the ticket.)
-  
-  b. This is a non-trivial task; I will complete it within X days.
-  
-  c. This task is out of scope or not high enough priority.
-  
-3. This is a bug report.
+If an issue is appropriately assigned, the assignee can provide valuable information to the person who reported the issue and to the EC by classifying it as follows:
 
-  a. I was able to confirm that the bug happens as described. It should be fixed within X days. (The assignee should not mark the ticket resolved until the fix is committed and merged.)
+1. This is a question. (This can include cases where a user is reporting a bug but actually they are just confused about the proper usage of a feature.)<br/>
+  a. The answer is... (Enter text to be forwarded to the user by Help Desk personnel.) (The assignee can then resolve (not close) the ticket.)<br/>
+  b. I will need to do more research to answer this. I will answer within X days.
+    (Also, it's helpful if you can indicate if this is a question that is likely to come up again and thus should be added by UE to the relevant documentation.)
+    
+2. This is a task.<br/>
+  a. I did the task. (The assignee can then close the ticket.)<br/>
+  b. This is a non-trivial task; I will complete it within X days.<br/>
+  c. This task is out of scope or not high enough priority. (The production lead and the EC can then verify that this is the case.)<br/>
   
+3. This is a bug report.<br/>
+  a. I was able to confirm that the bug happens as described. It should be fixed within X days. (The assignee should not close the ticket until the fix is committed and merged.)<br/>
   b. I could not reproduce the problem, and need more information.
 
-4. This is a request for an improvement or new feature.
-
-  a. This was easy to do so I did it. (The assignee can then resolve (not close) the ticket.)
-  
-  b. It is on our roadmap and will likely be included in release X.
-  
-  c. It's not on our roadmap, but it's a good idea, so EC should consider adding it to our list.
-  
+4. This is a request for an improvement or new feature.<br/>
+  a. This was easy to do so I did it. (Once the fix is committed, merged and deployed, the assignee can close the ticket.)<br/>
+  b. It is on our roadmap and will likely be included in release X.<br/>
+  c. It's not on our roadmap, but it's a good idea, so EC should consider adding it to our list.<br/>
   d. It's not on our roadmap and shouldn't be.
 
-UE will send the EC a monthly report about JIRA performance.
+An important part of the JIRA workflow is to return to the ticket when the issue is fixed, add an appropriate comment, and (if you think the issue is completely fixed and deployed, and the reporter should be able to see the new behavior) close the ticket.
```

Comment:
I would replace this whole paragraph just by "Don't forget to close your tickets when done!". Too wordy...